Content.Client/GameObjects/Components/HUD/Hotbar/HotbarComponent.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System.Collections.Generic;
+using Content.Client.GameObjects.EntitySystems;
+using Content.Client.UserInterface;
+using Robust.Client.GameObjects;
+using Robust.Client.UserInterface.Controls;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+using static Robust.Client.UserInterface.Controls.BaseButton;
+using static Robust.Shared.Input.PointerInputCmdHandler;
+
+namespace Content.Client.GameObjects.Components.HUD.Hotbar
+{
+    [RegisterComponent]
+    public class HotbarComponent : Component
+    {
+#pragma warning disable 649
+        [Dependency] private readonly IGameHud _gameHud;
+        [Dependency] private readonly IHotbarManager _hotbarManager;
+#pragma warning restore 649
+
+        public override string Name => "Hotbar";
+
+        private HotbarGui _hotbarGui;
+        private AbilityMenu _abilityMenu;
+        private Ability _menuAbilitySelected;
+
+        private List<Ability> _abilities;
+        private List<Ability> _hotbar;
```

Comment:
I think this could reasonably be an array. It wouldn't make much actual difference, but it might better communicate that the hotbar is fixed-size, and shouldn't be added to or taken from.

Reply:
I've designed most of it thinking we could have resizable hotbars, but if it's not needed, I can change it to an array.